lms/models/application_instance.py

```diff
@@ -53,6 +53,14 @@ class ApplicationInstance(BASE):
         "GroupInfo", back_populates="application_instance"
     )
 
+    def decrypted_developer_secret(self, aes_secret):
+        if self.developer_secret is None:
+            return None
+
+        cipher = AES.new(aes_secret, AES.MODE_CFB, self.aes_cipher_iv)
+
+        return cipher.decrypt(self.developer_secret)
```

Comment:
Moved `ApplicationInstanceGetter.developer_secret()` into the model. It has to be a method rather than a property because it needs `request.registry.settings["aes_secret"]` as an argument and, unlike services, models don't have factories to inject settings.